### PR TITLE
Allow javascript modules to also have template literals minified

### PR DESCRIPTION
**Pull request by XsnTrcK**
At the moment due to defaults in meriyah's parse method javascript with `import` statements are not parsed causing the template literals to not be minified.

Updated parse call to always allow parsing of javascript with `import` statements

### CODE REVIEW

**Comment by XsnTrcK** [comment]
@maxmilton Thoughts?

**Comment by maxmilton** [comment]
Sorry I've been travelling for a while and only just had a bit of time for my projects.

Looks good, however is there any big downside to using `module: true` so modules are parsed all the time regardless? I suspect the performance overhead would be minimal anyway + less options is always nice.

**Comment by XsnTrcK** [comment]
> Sorry I've been travelling for a while and only just had a bit of time for my projects.
> 
> Looks good, however is there any notable downside to using `module: true` so modules are parsed all the time regardless? I suspect the performance overhead would be minimal anyway + less options is always nice.

From my investigation doesn't look like there is any major noted downsides to setting `module: true`. Mainly added as a configuration in case it was intentional. Will update to just set `module: true` and revert changes to MinifyOptions and README

**Comment by XsnTrcK** [comment]
@maxmilton PR updated to just set `module: true` and add test to verify module javascript can be minified

**Review by maxmilton — APPROVED** [review]
Looks good :)

**Comment by maxmilton** [comment]
Sorry it took a while but your change has finally been publish in https://github.com/maxmilton/esbuild-minify-templates/releases/tag/v0.8.0

File: src/index.ts [hunk 1]
```diff
@@ -36,6 +37,7 @@ export function minify(code: string, opts: MinifyOptions = {}): MagicString {
     next: true,
     loc: true,
     ranges: true,
+    module: opts.allowModules ?? false,
```
[the remark this answers is not in the document]
Comment (XsnTrcK) [reply src/index.ts]: No reason. I was mostly just making sure this lib was handling the property properly. Could pass along undefined as it is treated as false in meriyah like we do here

**Pull request merged**